pyporscheconnectapi/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyporscheconnectapi/cli.py,sha256=Z4Vaz4SJ1ofqhXRUm98lERh7j-Tm0iXEAJbWaA7zmEE,10891
pyporscheconnectapi/client.py,sha256=m2ooVUaNzqfEBfRWYU7OPtRC6Poq78bVP0FCxkDP5B8,19073
pyporscheconnectapi/connection.py,sha256=2RyyngTnv-WTZW7KSwQ_9BEnm2Y2uZ3x9erlIQBwNLs,10073
pyporscheconnectapi/exceptions.py,sha256=Ez_D_TsafBd59LRIKRvGFv_6W9Ia425QCaClwXEke2w,1539
pyporscheconnectapi-0.0.19.dist-info/LICENSE,sha256=3wo4ylCH88NDNuSoNFaeaNCdzpQDloqsIUDQMg5FWec,1071
pyporscheconnectapi-0.0.19.dist-info/METADATA,sha256=BbE4IPEcGJvES8J7A_R6avNMXRPQLDJc5H7wQzjHcUY,512
pyporscheconnectapi-0.0.19.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
pyporscheconnectapi-0.0.19.dist-info/entry_points.txt,sha256=cq4djPuFhhopjtdIGqJ6w6vXOb6g65uLQSHnl7HAjfs,60
pyporscheconnectapi-0.0.19.dist-info/top_level.txt,sha256=YXSR3DO6I6QQ0lEQOvTts5BFIGsfOw_WtdFlD8BN4Yw,20
pyporscheconnectapi-0.0.19.dist-info/RECORD,,
